Extension/src/LanguageServer/configurations.ts

```diff
@@ -624,6 +624,21 @@ export class CppProperties {
         }, () => { }).catch(logAndReturn.undefined);
     }
 
+    public async updateCompilerPath(path: string): Promise<void> {
```

Comment:
It seems like this should be renamed to updateCompilerPathIfSet to avoid someone accidentally calling it when it's not set?